src/libs/actions/BankAccounts.ts

```diff
@@ -200,6 +201,13 @@ function addPersonalBankAccount(account: PlaidBankAccount) {
             },
         ],
     };
+    if (currentDefaultBankAccount?.methodID) {
+        onyxData.successData?.push({
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.BANK_ACCOUNT_LIST,
+            value: {[currentDefaultBankAccount.methodID]: {isDefault: false}},
+        });
+    }
```

Comment:
NAB. Can you please move this to `successData`

```js
            ...(currentDefaultBankAccount?.methodID
                ? [
                      {
                          onyxMethod: Onyx.METHOD.MERGE,
                          key: ONYXKEYS.BANK_ACCOUNT_LIST,
                          value: {[currentDefaultBankAccount.methodID]: {isDefault: false}},
                      },
                  ]
                : []),
```

Reply:
OK thanks. I will wait for the [discussion](https://github.com/Expensify/App/pull/35523?notification_referrer_id=NT_kwDOBbN2dLM5MjkzMzUwNzgzOjk1NjQ3MzQ4#issuecomment-1925809539) to conclude, if necessary I will 'push' these changes later.